.gitignore

```diff
@@ -207,3 +207,10 @@ fabric.properties
 
 # Ignore backups
 backups/
+
+# Ignore VS Code config file
+.vscode/*
+ !.vscode/settings.json
```

Comment:
Sorry, that is not what I meant, It is already there and you must remove the lines starting with `!.vscode[...]`